scripts/utils/createChangesetQuestions.ts

```diff
@@ -0,0 +1,41 @@
+const config = {
+  type: "table",
+  name: "packages",
+  message: "Choose changed package and  a version to upgrade",
```

Comment:
There is an extra blank space between 'package and' and 'a version'
```suggestion
  message: "Choose changed package and a version to upgrade",
```